README.md

```diff
@@ -61,66 +107,49 @@ Below is the flow of PSF with scripting support.
 The flow from beginning to end is [Starting Script] -> [Monitor] -> [Packaged exe] -> [Ending script]
 
 
-### Visual representation of the flow
-<pre>
-+------------+ Success, or  +-----------------+              +-----------------+
-|  Starting  | Script error |   Monitor and   |   Success    |      Ending     |
-|   Script   |------------->|   Packaged exe  |------------> |      Script     |
-+------------+              +-----------------+              +-----------------+
-     | Create                        | Create                         | Create
-     | Process                       | Process                        | Process
-     | Error                         | Error                          | Error
-     V                               V                                V
-+-------------+             +-----------------+   Create Process +----------------+
-| Throw error |             |    Run ending   |       Error      |   Throw error  | 
-|  and exit   |             |     script      |----------------->|    and exit.   |
-+-------------+             +-----------------+                  +----------------+
-                                    | 
-                                    | Success
-                                    |
-                                    V
-                             +--------------+
-                             |  Throw error |
-                             |   and exit   |
-                             +--------------+
-</pre>
-
 ## Sample configuration
-Here is a sample configuration using two different exes.
+Here is a sample configuration showing all possible options for each script.
 <pre>
     {
   "applications": [
     {
-      "id": "Sample",
-      "executable": "Sample.exe",
-      "workingDirectory": "",
-	  "startScript":
-	  {
-		"scriptPath": "RunMePlease.ps1",
-		"scriptArguments": "ThisIsMe.txt",
-		"runInVirtualEnvironment": true
-	  },
-	  "endScript":
-	  {
-		"scriptPath": "RunMeAfter.ps1",
-		"scriptArguments": "ThisIsMe.txt"
-	  }
+		"id": "Sample",
+		"executable": "Sample.exe",
+		"workingDirectory": "",
+		"stopOnScriptError": false
+		"startScript":
+		{
+			"wait": true,
+			"runOnce": false,
+			"timeout": 20,
+			"showWindow": true,
+			"scriptPath": "RunMePlease.ps1",
+			"scriptArguments": "ThisIsMe.txt",

```

Comment:
Why are arguments a txt file?

Reply:
Changed to remove ambiguity.  The text file was the argument.